docs/setup_guides/cloud.rst

```diff
@@ -190,156 +147,59 @@ you can do so by running::
-                      add=['SYS_ADMIN']
-                  )
-              )
-              return pod
-          c.KubeSpawner.modify_pod_hook = modify_pod_hook
-          c.JupyterHub.logo_file = '/usr/local/share/jupyter/hub/static/custom/images/logo.png'
-          c.JupyterHub.template_paths = ['/usr/local/share/jupyter/hub/custom_templates/',
-                                        '/usr/local/share/jupyter/hub/templates/']
-      image:
-        name: jupyterhub/k8s-hub
-        tag: v0.6
-      extraVolumes:
-        - name: custom-templates
-          gitRepo:
-            repository: "https://github.com/pangeo-data/pangeo-custom-jupyterhub-templates.git"
-            revision: "b09721bb1a1248dc115730d3c8a791600eae257e"
-      extraVolumeMounts:
-        - mountPath: /usr/local/share/jupyter/hub/custom_templates
-          name: custom-templates
-          subPath: "pangeo-custom-jupyterhub-templates/templates"
-        - mountPath: /usr/local/share/jupyter/hub/static/custom
-          name: custom-templates
-          subPath: "pangeo-custom-jupyterhub-templates/assets"
-
-    cull:
-      enabled: true
-      users: false
-      timeout: 1200
-      every: 600
-
-    # this section specifies the IP address for pangeo.pydata.org
-    # remove or change for a custom cluster
-    proxy:
-      service:
-        loadBalancerIP: 35.224.8.169
+  - EXTRA_PIP_PACKAGES: for adding some python modules to your user environment.
+  - GCSFUSE_BUCKET: for mounting some google cloud storage bucket as a standard
+    file system.
+
+The other file is
+` ``secret_config.yml`` <https://github.com/pangeo-data/pangeo/blob/master/gce/setup-guide/secret_config.yaml>`_,
+which specifies cluster specific encryption tokens.
+
+The jupyterhub proxy secret token is a random hash, which you can generate as follows:
 
-The other file is ``secret_config.yaml``, which specifies cluster specific
-encryption tokens. The jupyterhub proxy secret token is just a random hash,
-which you can generate as follows.
 
 .. code-block:: bash
 
   $ openssl rand -hex 32
 
-Pangeo.pydata.org uses `GitHub OAuth Callback
-<https://help.github.com/enterprise/2.13/admin/guides/user-management/using-github-oauth/>`_,
-(or `GitHub OAuth for developer <https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/>`_)
-to authenticate users. The ``clientSecret`` token needs to be obtained via
-github.
+Replace `<SECRET>` in the `secretToken` section with the output of this (note:
+you will stilll need the quotes around this value)
 
-This authentication method needs an IP or domain name to work, the IP you've
-reserved above and put in jupyter_config.yaml if you don't have a domain name
-yet (just put the IP in place of pangeo.pydata.org domain name).
 
-Alternatively, you can also change authentication method, see
-`Zero to Jupyterhub`_ guide for more information on that.
+Pangeo.pydata.org uses `GitHub OAuth Callback`_, (or `GitHub OAuth for developers`_)
```

Comment:
pangeo.pydata.org no longer exists :)

Perhaps we can just say "Most pangeo deployments us Github..."